.buildkite/pipeline.yml

```diff
@@ -39,3 +39,21 @@ steps:
           propagate-environment: true
           volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
+
+  - name: "VTAdmin Test"
+    command:
+    - apk add g++ make bash curl mysql mysql-client chromium
```

Comment:
This line is different from the other test setup lines. It is also installing chromium browser. We need this to run it in a headless mode to download the VTAdmin web pages. We can't use `curl` (which we use for the API pages) or `wget` because they do not support JavaScript, so a headless browser was the easiest solution.